src/LiveComponent/src/Util/LiveAttributesCollection.php

```diff
@@ -99,6 +100,8 @@ public function setBrowserEventsToDispatch(array $browserEventsToDispatch): void
 
     private function escapeAttribute(string $value): string
     {
-        return twig_escape_filter($this->twig, $value, 'html_attr');
+        $escapeExtension = new EscaperExtension();
+
+        return (string) $escapeExtension->escape($this->twig, $value, 'html_attr');
```

Comment:
What about this?
```suggestion
        return (string) EscaperExtension::escape($this->twig, $value, 'html_attr');
```

Since `EscaperExtension::escape` is static https://github.com/twigphp/Twig/blob/3.x/src/Extension/EscaperExtension.php#L182

Reply:
Good catch! I miss that the escape function was static